.idea/runConfigurations/Artemis__Server__LocalVC___LocalCI_.xml

```diff
@@ -1,6 +1,6 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Artemis (Server, LocalVC &amp; LocalCI)" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
-    <option name="ACTIVE_PROFILES" value="dev,jira,localci,localvc,artemis,scheduling,local" />
+    <option name="ACTIVE_PROFILES" value="dev,localci,localvc,artemis,scheduling,local" />
```

Comment:
Is this an intentional change for this PR?

Reply:
Yes. I decided to have the setup without Jira as the default and added into my documentation that you need to add "jira" to the profiles if you want to use it.

Reply:
Thank you for the clarification 👍